docs/marine-book/marine-runtime/configuration-file.md

```diff
@@ -23,11 +25,10 @@ A more complex example of a service configuration file is shown below:
 
 ```toml
 modules_dir = "artifacts/"
+total_memory_limit = "10 MiB"
 
 [[module]]
     name = "effector"
-    mem_pages_count = 1
-    max_heap_size = "1 KiB"
```

Comment:
so these ones are completely deprecated? I should deprecate them in CLI as well then

Reply:
Yes, they are